### PR TITLE
First Draft GraphQL

### PR DESCRIPTION
**Pull request by peoplenarthax**
Well, let's kickstart this graphql schema. Some parts like the N-M relationships (clubs to players, players to matches...) are a bit confusing. But the rest I think contains the minimum amount of info

### CODE REVIEW

File: src/graphql/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    """A club owner must be a claimed player"""
+    owner: Player!
+}
+
+type Boardgame {
+    type: GAME_TYPE!
+    id: ID!
+    name: String!
+    """ Link to boardgame characteristics (e.g. boardgamegeek) """
+    url: String
```
Comment (beeequeue) [inline src/graphql/schema.graphql:43]: Having a `rulebook` pdf url field sounds like a good idea as well. They are provided by the creators (e.g. [Scythe's official rulebooks](https://app.box.com/s/rj3jrw0rab2uiz02up89kbant5g8ew1p))

Comment (peoplenarthax) [reply src/graphql/schema.graphql:43]: absolutely, I will add it

**Review by beeequeue — APPROVED** [review]
We'll use this as a template when creating the models and such, as we want the code to generate the schema for us when running the program

**Comment by peoplenarthax** [comment]
Yes, It was to agree on some points for the overall schema, next step will be to create the models for this entities!

**Pull request merged**